packages/injected/src/wallets.ts

```diff
@@ -445,19 +445,17 @@ const tokenary: InjectedWalletModule = {
 
 const tally: InjectedWalletModule = {
   label: ProviderLabel.Tally,
-  injectedNamespace: InjectedNameSpace.Ethereum,
+  injectedNamespace: InjectedNameSpace.Tally,
   checkProviderIdentity: ({ provider }) =>
     !!provider && !!provider[ProviderIdentityFlag.Tally],
   getIcon: async () => (await import('./icons/tallywallet.js')).default,
   getInterface: async () => {
-    const provider = createEIP1193Provider(window.ethereum, {
-      eth_chainId: ({ baseRequest }) => baseRequest({ method: 'eth_chainId' }).then(id => `0x${parseInt(id).toString(16)}`),
```

Comment:
We have landed a fix for this [in this PR](https://github.com/tallycash/extension/pull/1247) which will be also in the next release. Is currently on main.

Reply:
@greg-nagy excellent,  will that change be in the upcoming release? ETA on the release?